packages/flutter_availability_data_interface/test/flutter_availability_data_interface_test.dart

```diff
@@ -1,11 +1,7 @@
-import "package:flutter_availability_data_interface/flutter_availability_data_interface.dart";
 import "package:flutter_test/flutter_test.dart";
 
 void main() {
-  test("adds one to input values", () {
-    var calculator = Calculator();
-    expect(calculator.addOne(2), 3);
-    expect(calculator.addOne(-7), -6);
-    expect(calculator.addOne(0), 1);
+  test("Expects tests to run", () {
```

Comment:
What is the point of changing this? Either replace it for a useful test or leave it.

Reply:
he can't leave it because he removed the Calculator